lib/fabrik/DirectorManager.js

```diff
@@ -59,14 +58,14 @@ class DirectorManager extends BaseManager {
   }
 
   get networkName() {
-    return this.subnet || infrastructure.segmentation.network_name || 'default';
+    return this.subnet || this.director.config.infrastructure.segmentation.network_name || 'default';
   }
 
   get resourcePools() {
     const networkName = this.networkName;
     const stemcell = this.stemcell;
-    return _.reduce(infrastructure.azs, (result, az) => {
-      _.forEach(infrastructure.vm_types, vm_type => {
+    return _.reduce(this.director.config.infrastructure.azs, (result, az) => {
+      _.forEach(this.director.config.infrastructure.vm_types, vm_type => {
```

Comment:
comments similar to [above](https://github.com/SAP/service-fabrik-broker/pull/7/files#r133368904)